### PR TITLE
Split workshops into 201 and 202

### PR DESCRIPTION
**Pull request by patricksimonian**
## Rationale

After taking a look through the 201 course, it was very apparent there is a lot of great content there. 
In my opinion, which contains several years of deliver course-formatted training, it is too much to focus on when doing a 'light touch' course.  Splitting the course into two more focused deliverables will:

- make it more easy to consume
- will ease the amount of background knowledge facilitators would need to deliver 201 (no strict patroni experience)

## Thoughts on 202

With the 201, the students will have more than enough experience to be able to standup their own workshops.  I suggest we provide guidance on standing up a 202 course on their own. That or, we turn it into a Devhub Journey

### CODE REVIEW

**Comment by jefkel** [comment]
I see the labs broken out, but were you also going to modify the presentation material to match?

I get dropping some of the additional labs/content for the 201 to make it more palatable with the time-frame and delivery type.

I think there is still value in presenting students with a specific problem and then asking them to solve it.. (otherwise, it's not training, it's just project work).  
Bite Sized Lab Ideas:
- Deploying a postgres multi-node DB
- Backing up a DB
- Validating DB Backups
etc...

**Comment by patricksimonian** [comment]
Yep I was going to break up the content to match the labs. This was going to be done by just doing some slight modifications to the content to match our 'touch points' schedule. How does that sound?

And I totally agree! That was my hope :) 

201 - Templating your Openshift Project
202 - Deploying a postgres multi-node DB
203 - Backing up a DB
        - Validating DB Backups

File: workshop-labs/docs/openshift202/setup.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Lab Setup
+Prior to working on labs, the Platform Services team will have created 4
```
Comment (ShellyXueHan) [inline workshop-labs/docs/openshift202/setup.md:2]: 3 projects

File: workshop-labs/docs/openshift202/setup.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - Navigate to the top right corner, select the drop-down from your username, and select `Copy Login Command`
+    - Paste the copied command into your terminal of choice
+
+```
+oc login https://console.pathfinder.gov.bc.ca:8443 --token=[token]
+```
+
+## Communication Resources
+The lab instructors will open up a dedicated RocketChat channel for sharing of code, content, or discussions for the duration of this class.
+Please provide the instructors with your RocketChat user id at the same time as your GitHub ID when starting the workshop.
```
Comment (ShellyXueHan) [inline workshop-labs/docs/openshift202/setup.md:34]: are you running the same setup script from 101? If so, let's get a more permanent link for that.

Comment (patricksimonian) [reply workshop-labs/docs/openshift202/setup.md:34]: which setup script the `workbench` one?

Comment (ShellyXueHan) [reply workshop-labs/docs/openshift202/setup.md:34]: yup. it takes id from a google sheet

**Comment by patricksimonian** [comment]
> Content changes to match reduction in lab work for the 201 course.

I think i've fixed what you were mentioning :)

File: workshop-labs/docs/openshift202/setup.md (hunk 2 above)
Comment (patricksimonian) [reply workshop-labs/docs/openshift202/setup.md:34]: is there a template?

Comment (ShellyXueHan) [reply workshop-labs/docs/openshift202/setup.md:34]: It's sourcing from a sheet, but could be anything.
- the script is here: https://github.com/BCDevOps/devops-platform-workshops/blob/patch/ocp101-2020-01-23/.openshift/setup-students.sh
- template for the google sheet: https://github.com/BCDevOps/devops-platform-workshops/blob/patch/ocp101-2020-01-23/101-materials/Faciliator-Handbook.md#instructor

**Comment by patricksimonian** [comment]
im going to merge this then work on combining clecios workshop patch so it all  makes sense together

**Comment by jefkel** [comment]
> 202 - Deploying a postgres multi-node DB
> 203 - Backing up a DB
> - Validating DB Backups

Probably need to dive in more to the labs/content if you're looking to break up the DB configuration.  I'd recommend 
202 - getting a multi-node DB up and running, tweak a configuration setting, validate failover scenarios, where to look for logs, events, etc.

203 - Backup/restore, advanced configuration changes, migrating storage.

**Pull request merged**